src/FixerConfiguration/FixerConfigurationResolver.php

```diff
@@ -58,6 +59,19 @@ public function resolve(array $options)
         foreach ($this->options as $option) {
             $name = $option->getName();
 
+            if ($option instanceof AliasedFixerOption) {
+                $alias = $option->getAlias();
+
+                if (array_key_exists($alias, $options)) {
+                    if (array_key_exists($name, $options)) {
```

Comment:
Please add the following comment before this line: `// @TODO on 2.12: trigger a deprecation notice and add a test for it`.